build/media-suite_compile.sh

```diff
@@ -2221,6 +2221,21 @@ if [[ $ffmpeg != no ]] && enabled libxevd &&
     do_checkIfExist
 fi
 
+_check=(bin-video/oapv_app_{enc,dec}.exe oapv/oapv{,_exports}.h liboapv.a oapv.pc)
+if [[ $ffmpeg != no ]] && enabled liboapv &&
+    do_vcs "$SOURCE_REPO_OPENAPV"; then
+    do_uninstall bin-video/liboapv.dll lib/liboapv.dll.a.dyn "${_check[@]}"
+    do_cmakeinstall video -DCMAKE_SYSTEM_NAME=Windows -DCMAKE_SYSTEM_PROCESSOR=x86_64
+    # no way to disable shared lib building in cmake
+    # move the libraries out from subfolder to make ffmpeg configure find it easier
+    mv -f "$LOCALDESTDIR"/lib/oapv/liboapv.a "$LOCALDESTDIR"/lib/liboapv.a
+    mv -f "$LOCALDESTDIR"/lib/oapv/import/liboapv.dll.a "$LOCALDESTDIR"/lib/liboapv.dll.a.dyn
```

Comment:
~~this can be covered by a hide_all_shared_libs or whatever the function is called~~

just noticed the special `/import/` part of the path, not sure what to make of it yet.